CHANGELOG.md

```diff
@@ -1,7 +1,3 @@
-## 3.1.1

```

Comment:
this should not be deleted, also add 3.1.2 patchnote

Reply:
Done